docs/src/pages/components/authenticator/index.page.mdx

```diff
@@ -729,18 +729,24 @@ If you'd like to update the primary color of your submit button you can override
 
 These variables are used in the theming of the Authenticator. You can override these values to update the look and feel.
 
+| Amplify CSS Variable                           | Default Value                             |
+| ---------------------------------------------- | ----------------------------------------- |
+| --amplify-components-alert-align-items         | center                                    |
+| --amplify-components-alert-justify-content     | space-between                             |
+| --amplify-components-alert-color               | var(--amplify-colors-font-primary)        |
+| --amplify-components-alert-background-color    | var(--amplify-colors-background-tertiary) |
+| --amplify-components-alert-padding-block       | var(--amplify-space-small)                |
+| --amplify-components-alert-padding-inline      | var(--amplify-space-medium)               |
+| --amplify-components-alert-icon-size           | var(--amplify-font-sizes-xl)              |
+| --amplify-components-alert-heading-font-size   | var(--amplify-font-sizes-medium)          |
+| --amplify-components-alert-heading-font-weight | var(--amplify-font-weights-bold)          |
+| --amplify-components-alert-info-color          | var(--amplify-colors-font-info)           |
+
+<details>
+  <summary>More...</summary>
```

Comment:
What do you think about using vertical [`ScrollView` ](https://ui.docs.amplify.aws/components/scrollview) instead? We've been using it for long code snippets, but curious how it looks with tables.

Reply:
We could use ScrollView within an Expander too maybe?

Reply:
Good idea 👍  Let me try it out